src/Core/Auth/Models/Api/Response/UserDecryptionOptions.cs

```diff
@@ -32,10 +32,14 @@ public UserDecryptionOptions() : base("userDecryptionOptions")
 public class TrustedDeviceUserDecryptionOption
 {
     public bool HasAdminApproval { get; }
+    public string? EncryptedPublicKey { get; }
+    public string? EncryptedUserKey { get; }
 
-    public TrustedDeviceUserDecryptionOption(bool hasAdminApproval)
+    public TrustedDeviceUserDecryptionOption(bool hasAdminApproval, string? encryptedPublicKey, string? encryptedUserKey)
     {
         HasAdminApproval = hasAdminApproval;
+        EncryptedPublicKey = encryptedPublicKey;
+        EncryptedUserKey = encryptedUserKey;
```

Comment:
**question/issue:** Doesn't this also need to contain a private key? Trusting a device requires three keys: 
- DevicePublicKey(UserKey)
- UserKey(DevicePublicKey)
- DeviceKey(DevicePrivateKey)

Reply:
*facepalm* 

Thank you for catching that!

Reply:
Yeah very good catch, I was working off a conversation with Jared and I knew I only needed 2/3 but I someone got it wrong even though our conversation had the correct ones.

Reply:
It happens sometimes. I read it and thought, "yep, this is all correct" even though we both had the conversation about the correct keys.